build.gradle.kts

```diff
@@ -65,7 +65,7 @@ dependencies {
   implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.7.0")
 
   // Thymeleaf
-  implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
+  implementation("org.springframework.boot:spring-boot-starter-thymeleaf:3.1.2")
```

Comment:
not sure you need to manage the version here, if you don't specify it then it'll default to a version that's in line with the wider spring boot.